static/templates/compose_control_buttons.hbs

```diff
@@ -1,6 +1,6 @@
 <input type="file" class="file_input notvisible pull-left" multiple />
 {{#if file_upload_enabled }}
-<a role="button" class="compose_control_button fa fa-paperclip notdisplayed" aria-label="{{t 'Attach files' }}" id="attach_files" tabindex=0 title="{{t 'Attach files' }}"></a>
+<a role="button" class="compose_control_button attach_files fa fa-paperclip notdisplayed" aria-label="{{t 'Attach files' }}" tabindex=0 title="{{t 'Attach files' }}"></a>
```

Comment:
Maybe `compose_upload_file` would be a better name for this?  

And probably the label should be "Upload files" too.

Reply:
Yeah.